en/docs/install-and-setup/deploying-wso2-api-manager/production-deployment-guidelines.md

```diff
@@ -71,120 +82,182 @@ Environment compatibility
-  ```|
-| CentOS  | Open a terminal and run the following command:                                                                                                                                                                                            
-                                                                                                                                                                                                                                             
-  ``` java                                                                                                                                                                                                                                   
-        sudo yum remove <PRODUCT_DISTRIBUTION_NAME>-x86_64                                                                                                                                                                                         
-  ```|
-
-#### Setting system properties
+| Ubuntu  | Open a terminal and run the following command: <br>````sudo apt-get purge <PRODUCT_DISTRIBUTION_NAME>````                                                       
+| CentOS  | Open a terminal and run the following command: <br>````sudo yum remove <PRODUCT_DISTRIBUTION_NAME>-x86_64 ````                                                                                                                                                                                                                                             
+                                                                                                                                                                                                                                  
+### Setting System Properties
 
 If you need to set additional system properties when the server starts, you can take the following approaches:
 
--   **Set the properties from a script** . Setting your system properties in the startup script (i.e. the `<PRODUCT_HOME>/bin/wso2server.` sh file), is ideal because it ensures that you set the properties every time you start the server. To avoid having to modify the script each time you upgrade, the best approach is to create your own startup script that wraps the WSO2 startup script and adds the properties you want to set, rather than editing the WSO2 startup script directly.
+-   **Set the properties from a script** . Setting your system properties in the product's startup script (wso2server.sh
+  for Linux and wso2server.bat for Windows), is ideal because it ensures that you set the properties every time you 
+  start the server. To avoid having to modify the script each time you upgrade, the best approach is to create your own startup script that wraps the WSO2 startup script and adds the properties you want to set, rather than editing the WSO2 startup script directly.
```

Comment:
To avoid modifying the script each time you upgrade, the best approach is to create your own startup script with the properties you want to set that wraps the WSO2 startup script, rather than editing the WSO2 startup script directly.

Reply:
This section was not changed by me. Only the formatting has been changed by the pr. BTW seems the existing sentence gives the exact meaning better, than what you suggested.